enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -693,23 +732,6 @@ revisit this design decision ahead of tech preview.
 
 **Resolution**: TBD for tech preview.
 
-#### Should we enable automated deployments?
-
-When the Ingress Operator installs OpenShift Service Mesh (OSSM) and creates a
-ServiceMeshControlPlane (SMCP) CR, OSSM creates an Envoy proxy named
-"istio-ingressgateway".  The cluster admin can then configure a Gateway CR that
-uses this Envoy proxy.
-
-Istio has a feature called [automated deployment](https://istio.io/latest/docs/tasks/traffic-management/ingress/gateway-api/#automated-deployment) whereby it creates a new
-Envoy proxy for a Gateway.
-
-_Should we use automated deployments?_
-
-To answer this question, we need to better understand the behavior and security
-implications of automated deployments.
-
-**Resolution**: TBD.
```

Comment:
I've been keeping open questions for context and adding the resolutions to record that we discussed them.  I recently explained my rationale for this approach and gave examples from other EPs here: https://github.com/openshift/enhancements/pull/1296#discussion_r1061556659